### ISSUE TITLE
Create Notification component for WhatsApp Textmebot API

### ISSUE THREAD

**Issue by apilaura** [louislam/uptime-kuma#4316]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Notification

### 🔖 Feature description

I need help to create a new Notification component based on the Textmebot Whatsapp API. It is a simple web API to send WhatsApp Messages. I want to send the notifications via whatsapp using this API.
API url: https://textmebot.com

### ✔️ Solution

I want to send the notifications via WhatsApp using TextMeBot API.

Create a new Notification component that will capture two parameters:
1) APIkey
2) Recipient Phone number.

Then call to the web API using the apikey, phone number and the standard message.

The API has this format:
http://api.textmebot.com/send.php?recipient=<phone_number>&apikey=<apikey>&text=<notification_message>

I can provide a free apikey and a $100 donation to the person who can help me.

### ❓ Alternatives

I have considered to modify some of the existing notification components and change the endpoint, parameters, etc... but I need help with the PR as I am not familiar with github methods.

**Comment by CommanderStorm** [comment]
We currently don't support templating in the url field.
See #1734 for further details. (not because of technical reasons, we just have not gotten around to this part)

Adding WhatsApp is tracked in #1816 

I am reading the website you linked, that you can request minor modifications via support@textmebot.com
I think allowing for text as a body is reasonable.

> but I need help with the PR as I am not familiar with github methods

What part are you stuck on? Note that our [contribution guide can be found here](https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md)

**Comment by apilaura** [comment]
The WhatsApp API already support POST requests and pass the parameters in the body.
Check here: https://documenter.getpostman.com/view/20714522/UzXLyxPD#8d96c754-fbe5-4a6b-b195-dc7ec2e473f1

Endpoint: `https://api.textmebot.com/send.php`
The POST body could contain:
```
{
"recipient":"+39xxxxxxxx",
"apikey":"my_apikey_yyyyyyy",
"text":"The monitoring alert here"
}
```

I will have a look at the contribution guide but basically I am completely new with github. I only know how to clone but have no experience with forks, branches, pull requests, commits, etc...

**Comment by CommanderStorm** [comment]
> The WhatsApp API already support POST requests and pass the parameters in the body

Great, so you can use the webhook to post there via the templating already implemented.
No modification/PR needed.

> I will have a look at the contribution guide

Honestly, I think using this provider for WhatsApp is not ideal. The official API is free for up to 1k Messages => using the API noted in https://github.com/louislam/uptime-kuma/issues/1816 is better from a contribution perspective

> I am completely new with github

If you come across any problems, feel free to comment. We have all been there ^^

**Comment by andrealvess** [comment]
my whatsapp for your help: +5583988925254

**Comment by CommanderStorm** [comment]
I am assuming this issue is resolved via the webhook notification provider.

FYI (for future readers):
We currently have 3 (2 of which to be released in `v2.0`) notification providers that support Whatsapp https://github.com/louislam/uptime-kuma/pulls?q=is%3Apr+Whatsapp
